app/views/layouts/application.html.erb

```diff
@@ -26,17 +26,15 @@ along with SSID.  If not, see <http://www.gnu.org/licenses/>.
   <header class="site_header">
     <div class="container d-flex flex-row w-100">
       <% unless user_signed_in?%>
-        <div class="w-20">
-          <%= image_tag("logo.png", height: "100", width: "150", alt: "NUS logo", class: "site_logo")%>
-        </div>
-        <h3 id="site-banner-login">Student Submissions Integrity Diagnosis System</h3>
-        <div class="site-login w-20">
-          <ul id="menu" <%= %Q{ class="admin"}.html_safe if user_signed_in? and current_user.is_admin %>>
-            <li <%= highlight_if_current new_user_session_path %>><%= link_to "Login", new_user_session_path %></li>
-            <li <%= highlight_if_current new_user_registration_path %>><%= link_to "Sign Up", new_user_registration_path%></li>
-          </ul>
-        </div>
+          <h3 id="site-banner-login"><a href='/' style="color:inherit; text-decoration:inherit">Student Submissions Integrity Diagnosis System</a></h3>
+          <div class="site-login w-20">
+            <ul id="menu" <%= %Q{ class="admin"}.html_safe if user_signed_in? and current_user.is_admin %>>
+              <li> <button type="button" style="background-color: transparent; color: #003d7c; height: 3rem; padding: 1rem" class="btn py-1" onclick="location.href='/login';">Log In</button> </li>
```

Comment:
Should the inline styling be extracted into a css class?

Reply:
I'll merge this version in first, @joshenx you can do refactoring on this afte the merge